compiler/packages/react-compiler-healthcheck/src/index.ts

```diff
@@ -39,7 +38,13 @@ async function main() {
     ],
   };
 
-  for (const path of await glob(src, globOptions)) {
+  let paths = [];
```

Comment:
Maybe use a Set here in case if there are duplicate paths 
```suggestion
  constant paths = new Set();
```